### ISSUE TITLE
Failed to start storage service after restarting it

### ISSUE THREAD

**Issue by life1347** [fission/fission#350]
Storage service failed to start with following error message because it doesn't check the existence of `/fission/fission-functions` before calling `CreateContainer`
```
panic: Error initializing storage: mkdir /fission/fission-functions: file exists
```

**Comment by soamvasani** [comment]
Going to keep this open for fixing it while properly using the stow API (i.e. use WalkContainers instead of constructing the container id) @life1347

**Comment by soamvasani** [comment]
fixed by #353
